torch/_torch_docs.py

```diff
@@ -9335,6 +9335,10 @@ def merge_dicts(*dicts):
 0-D and 1-D tensors are returned as is. When input is a 2-D tensor this
 is equivalent to ``transpose(input, 0, 1)``.
 
+.. seealso::
+
+        :func:`torch.transpose`.
```

Comment:
add a half sentence explaining the difference/similarity

Reply:
Added, will update the screenshots once the preview gets generated.